### DISCUSSION TITLE
[Connector-Request] Pinata

### DISCUSSION THREAD

**Discussion by aazam-gh** [airbytehq/airbyte#47337] [New Connector Request]
Topic
This is a suggestion to create a connector for Pinata using the UI Connector Builder. The website for the connector is 
https://pinata.cloud/

Relevant information
https://docs.pinata.cloud/quickstart

**Comment by aazam-gh** [comment]
@marcosmarxm I would like to take this up

**Comment by marcosmarxm** [comment]
I don't think Pinata is a good example to create connector using Builder because you won't be able to fetch the actual data from files only list them.
